tools/setup-envtest/README.md

```diff
@@ -40,6 +41,16 @@ setup-envtest use -i --use-env
 
 # sideload a pre-downloaded tarball as Kubernetes 1.16.2 into our store
 setup-envtest sideload 1.16.2 < downloaded-envtest.tar.gz
+
+# Per default envtest binaries are downloaded from: 
+# https://raw.githubusercontent.com/kubernetes-sigs/controller-tools/master/envtest-releases.yaml
+# To download from a custom index use the following:
+setup-envtest use --index https://custom.com/envtest-releases.yaml
+
+# To download from the kubebuilder-tools GCS bucket: (default behavior before v0.18)
+# Note: This is a Google-owned bucket and it might be shutdown at any time
+# see: https://github.com/kubernetes/k8s.io/issues/2647#event-12439345373
+setup-envtest use --use-gcs
```

Comment:
do we need to keep the old/ current way?
It will die and we cannot ensure that will be available

Reply:
I would preserve it for now but deprecate it clearly.

For the next few months while the kubebuilder GCS bucket is still available it might be a good fallback if there is anything wrong with the new implementation

Reply:
I'll do an iteration on deprecating it everywhere :)